anvil/src/shell/x11.rs

```diff
@@ -206,9 +211,7 @@ impl<BackendData: Backend> XwmHandler for CalloopData<BackendData> {
     }
 
     fn resize_request(&mut self, _xwm: XwmId, window: X11Surface, _button: u32, edges: X11ResizeEdge) {
-        let seat = &self.state.seat; // luckily anvil only supports one seat anyway...
```

Comment:
This comment is still very much true, if we store a `PointerHandle` in the state.